pyfr/solvers/euler/inters.py

```diff
@@ -73,11 +73,13 @@ class EulerCharRiemInvBCInters(EulerBaseBCInters):
     type = 'char-riem-inv'
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+        super().__init__(be, lhs, elemap, cfgsect, cfg)
```

Comment:
These looks undefined to me which implies that this has not been tested with Euler.